src/models/Uploads.php

```diff
@@ -141,6 +141,28 @@ public function createFromString(string $fileType, string $realName, string $con
         $MakeThumbnail->makeThumb();
     }
 
+    /**
```

Comment:
I think it would be better to use the already existing replace() function, as it will do ownership checks and update the timestamp (also, less code is better).